question-service/app/crud/questions.py

```diff
@@ -28,3 +28,28 @@ async def get_all_questions() -> QuestionCollection:
 async def get_question_by_id(question_id: str) -> QuestionModel:
     existing_question = await question_collection.find_one({"_id": ObjectId(question_id)})
     return existing_question
+
+async def delete_question(question_id: str):
```

Comment:
Should have the deletion logic in a separate branch so that in the case where deletion PR is not approved for some reason, this PR can still proceed.